pkg/apis/contrail/v1alpha1/command_types.go

```diff
@@ -42,7 +41,7 @@ type CommandConfiguration struct {
 // +k8s:openapi-gen=true
 type CommandStatus struct {
 	Active       bool                `json:"active,omitempty"`
```

Comment:
Just an improvement:
 
Let's use this generic status struct
https://github.com/Juniper/contrail-operator/blob/a1010f2ef7e636b020e90b58b0927d9549de97f9/pkg/apis/contrail/v1alpha1/memcached_types.go#L15

and some nice column prints

https://github.com/Juniper/contrail-operator/blob/a1010f2ef7e636b020e90b58b0927d9549de97f9/pkg/apis/contrail/v1alpha1/memcached_types.go#L33

Reply:
Done